### PR TITLE
Addition of Drain Info Endpoint for Support Debugging

### PR DESCRIPTION
**Pull request by husobee**
### Type of change ( select one )

- [x] Product feature
- [ ] Bug fix
- [ ] Performance improvement
- [ ] Refactor
- [ ] Other

### Tested Environments

- [ ] Development
- [ ] Staging
- [ ] Production

### Before submitting this PR:

- [x] Does your code build cleanly without any errors or warnings?
- [ ] Have you used auto closing keywords?
- [x] Have you added tests for new functionality?
- [x] Have validated query efficiency for new database queries?
- [x] Have documented new functionality in README or in comments?
- [x] Have you squashed all intermediate commits?
- [x] Is there a clear title that explains what the PR does?
- [x] Have you used intuitive function, variable and other naming?
- [ ] Have you requested security / privacy review if needed
- [x] Have you performed a self review of this PR?

### CODE REVIEW

**Review by keur — COMMENTED** [review]
Code LGTM but adding tests for the merging and batching logic in GetCustodianDrainInfo would be good

File: utils/outputs/meta.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package outputs
```
Comment (yachtcaptain23) [inline utils/outputs/meta.go:1]: (1) My gut feeling is that `utils/outputs/meta.go` isn't the best place to start stuffing HTTP responses. Perhaps it should be under the controller like `controller/responses/`?
(2) I think this file should be broken up.

Comment (husobee) [reply utils/outputs/meta.go:1]: utils/outputs was meant for outputs to http controllers.  I can rename if it does not make sense.

Comment (husobee) [reply utils/outputs/meta.go]: resolved: https://github.com/brave-intl/bat-go/pull/820/commits/2f953cdeb8596dd694927d6d6006da49284aa038

**Review by keur — DISMISSED** [review]
LGTM

File: promotion/datastore.go [hunk 2]
```diff
@@ -680,6 +684,93 @@ func (pg *Postgres) SaveClaimCreds(creds *ClaimCreds) error {
...
+	split_part(credentials->0->>'issuer',':',1) as promotion_id,
+	completed_at,
+	json_array_length(credentials)*0.25 as value,
+	case when erred then 'errored' else 'succeeded' end as state,
+	errcode,
+	transaction_id
+from
+	claim_drain
+where
+	wallet_id = $1
```
Comment (yachtcaptain23) [inline promotion/datastore.go:720]: I don't think this is indexed (I'm just grepping through the migrations so I could be wrong, wish we had a schema file)

Comment (husobee) [reply promotion/datastore.go:720]: good catch.  resolved: https://github.com/brave-intl/bat-go/pull/820/commits/4160c96c3f6f88ec414f630e96ffb51bb9310455

**Review by yachtcaptain23 — APPROVED** [review]
LG2M. Thanks for the all the changes!

**Pull request merged**